lochness/mindlamp/__init__.py

```diff
@@ -144,7 +247,8 @@ def get_study_lamp(lamp: LAMP) -> Tuple[str, str]:
     return study_obj['id'], study_obj['name']
 
 
-def get_participants_lamp(lamp: LAMP, study_id: str) -> List[str]:
+def get_participants_lamp(lamp: LAMP, study_id: str,
+                          from_ts: str = None, to_ts: str = None) -> List[str]:
```

Comment:
`from_ts` and `to_ts` are unused. Do we need them here?

Reply:
Thanks for catching this. Not used- I'll remove it.

Reply:
aca77b9